src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -24,7 +24,8 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        String login = req.getParameter("login");
+        Driver driver = new Driver(name, licenseNumber,login);
```

Comment:
```suggestion
        Driver driver = new Driver(name, licenseNumber, login, password);
```
also need to add password when you kind of registering new driver to DB

Reply:
Also add after String login : String password = req.getParameter("password");